apps/crn-frontend/src/tags/Routes.tsx

```diff
@@ -11,6 +12,11 @@ const Routes: React.FC<Record<string, never>> = () => {
   const { client } = useAlgolia();
   const { tags, setTags } = useSearch();
 
+  const entities: Array<keyof EntityResponses['crn']> = [
```

Comment:
Maybe this should be exported from model or something? It doesn't feel like a prop unless it changes because of a filter i guess